crates/node/src/catchup/batch.rs

```diff
@@ -5,13 +5,12 @@ use eyre::Result as EyreResult;
 use futures_util::SinkExt;
 use serde_json::to_vec as to_json_vec;
 
-use crate::types::{
-    CatchupError, CatchupStreamMessage, CatchupTransactionsBatch, TransactionWithStatus,
-};
+use crate::types::{ActionMessage, CatchupActionsBatch, CatchupError, CatchupStreamMessage};
 
+// TODO: Should we be using this somewhere in the CRDT approach, or not?
```

Comment:
@miraclx just want to make sure this wasn't missed.

Reply:
Yeah, this makes sense to retain.